exports/exports.go

```diff
@@ -68,17 +69,32 @@ func (e *Export) PostExport(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if len(apiExport.Sources) == 0 {
+		logger.Errorw("no sources provided", "error", err)
+		BadRequestError(w, "no sources provided")
+		return
+	}
+
+	cfg := config.Get()
+
+	var exportableApplications = config.ConvertConfigtoInternal(cfg.ExportableApplications)
+	
+	for _, source:= range apiExport.Sources{
```

Comment:
Moving the `for` loop into the `verifyExportableApplication()` would make this a bit easier to read.